python/foglamp/services/core/api/snapshot_config.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
```

Comment:
We should have `core/api/snapshot/tables.py` and map the routes for supported table names [configuration, schedules].

Reply:
No, the snapshot will work with any table, so there is no need to do this.

Reply:
You mean we should not restrict to `[configuration, schedules]` tables?

Reply:
Correct. The only table you can not snapshot is the readings. That is because there is no API to do it in the storage layer.

Reply:
@MarkRiddoch now there is some mismatch here vis-a-vis specs. Per specs mentioned in FOGL-2652 and FOGL-2654, we are required to define table specific routes for the tables configuration and schedules. Meaning that table snapshot route definitions are not generic and are bound to a specific table. If we want routes for table snapshot to be generic, then we must define routes accordingly. Something like:

POST /foglamp/snapshot/table/{table_name}
GET /foglamp/snapshot/table/{table_name}
GET /foglamp/snapshot/tables
PUT /foglamp/snapshot/table/{table_name}/id/{id}
DELETE /foglamp/snapshot/table/{table_name}/id/{id}

Am I missing something?